lpa-submissions-e2e-tests/cypress/support/common/validateFileUploadErrorMessage.js

```diff
@@ -0,0 +1,27 @@
+/// <reference types = "Cypress"/>
```

Comment:
Can this be removed.

Reply:
This is currently consistent with what we have on the project. I can take this as a separate task to remove it